packages/caver-transaction/src/transactionHelper/transactionHelper.js

```diff
@@ -132,19 +133,23 @@ const refineSignatures = (sigArray, isLegacy = false) => {
     const set = new Set()
     let result = []
 
-    const arrayOfSignatures = !_.isArray(sigArray[0]) ? [sigArray] : sigArray
-
+    let arrayOfSignatures = sigArray
```

Comment:
This function needs to modify function description. :)

Reply:
Thank you :). I modified the description